io.github.Andrettin.Wyrmsun.yml

```
@@ -0,0 +1,88 @@
+app-id: io.github.Andrettin.Wyrmsun
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: wyrmsun
+rename-icon: wyrmsun
+rename-desktop-file: wyrmsun.desktop
+rename-appdata-file: wyrmsun.appdata.xml
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --persist=.stratagus
+
+modules:
+  - shared-modules/lua5.1/lua-5.1.5.json
+  - shared-modules/physfs/physfs.json
+
+  - name: sqlite3
+    config-opts:
+      - --enable-threadsafe
+      - --enable-threads-override-locks
+    build-options:
+      - cflags: -DSQLITE_ENABLE_UNLOCK_NOTIFY=1
```

```suggestion
    build-options:
        cflags: -DSQLITE_ENABLE_UNLOCK_NOTIFY=1
```

flatpak builder actually print a warning. I know it's yaml, it's confusing.